dkeyserver/Enclave/enclave.cpp

```diff
@@ -494,3 +494,278 @@ int sgx_set_up_tls_server(char *server_port)
         EVP_PKEY_free(pkey);
     return ret;
 }
+
+unsigned long inet_addr2(const char *str)
+{
+    unsigned long lHost = 0;
+    char *pLong = (char *)&lHost;
+    char *p = (char *)str;
+    while (p)
+    {
+        *pLong++ = atoi(p);
+        p = strchr(p, '.');
+        if (p)
+            ++p;
+    }
+    return lHost;
+}
+
+int connect(int sockfd, const struct sockaddr *servaddr, socklen_t addrlen)
+{
+    int ret = -1;
+
+    if (ocall_connect(&ret, sockfd, servaddr, addrlen) == SGX_SUCCESS)
+        return ret;
+
+    return -1;
+}
+
+int create_socket(const char *server_name, uint16_t server_port)
+{
+    int sockfd = -1;
+    struct sockaddr_in dest_sock;
+    int res = -1;
```

Comment:
suggest to rename the res -> ret and check all the functions

Reply:
done.

Reply:
when you reply as done, have you pushed your changes to this patch?

Reply:
just locally changed. Shall I push the changes after handling all the comment?

Reply:
yes, after you pushed the new change, then you can mark it as "done"